FundParser/FundParser.DAL.Tests/UnitOfWorkTests.cs

```diff
@@ -12,7 +13,7 @@ public class UnitOfWorkTests
         public class GetByIDTests : UnitOfWorkTestsBase
         {
             [Test]
-            public async Task GetByID_NonExistingEntity_ReturnsNull()
+            public async Task GetByID_NonExistingEntity_ThrowsException()
```

Comment:
Again, both of these tests should be changed to use constraint asserts if it is modified in the previously mentioned classes.